pkg/apis/apps/v1alpha1/apimanager_types.go

```diff
@@ -144,8 +179,23 @@ type SystemSpec struct {
 
 	// +optional
 	DatabaseSpec *SystemDatabaseSpec `json:"database,omitempty"`
+
+	SystemAppSpec     *SystemAppSpec     `json:"systemAppSpec,omitempty"`
+	SystemSidekiqSpec *SystemSidekiqSpec `json:"systemSidekiqSpec,omitempty"`
+}
+
+type SystemAppSpec struct {
+	// +optional
+	Replicas *int64 `json:"replicas,omitempty"`
+}
+
+type SystemSidekiqSpec struct {
+	// +optional
+	Replicas *int64 `json:"replicas,omitempty"`
 }
 
+// TODO could memcached replicas be configurable?
```

Comment:
I'm not sure at this moment even though that memcached is a "bd" maybe we could scale the replicas. To be conservative I've not make it configurable

Reply:
I would not scale memcached pod replica. Scaling DB is not straightforward, should be done with more care.